api.py

```diff
@@ -22,9 +22,16 @@ async def status():
 
 logger = get_logger()
 
-AJV_HOST = os.getenv("AJV_HOST", "localhost")
+AJV_VALIDATOR_SCHEME = os.getenv("AJV_VALIDATOR_SCHEME", "http")
```

Comment:
How about this?

AJV_VALIDATOR_SCHEME = os.getenv("AJV_VALIDATOR_SCHEME", "http")
AJV_VALIDATOR_HOST = os.getenv("AJV_VALIDATOR_HOST", "localhost")
AJV_VALIDATOR_PORT = os.getenv("AJV_VALIDATOR_PORT", "5002")
AJV_VALIDATOR_URL  = os.getenv("AJV_VALIDATOR_URL", "")

if AJV_VALIDATOR_URL== "":
    AJV_VALIDATOR_URL = (
        f"{AJV_VALIDATOR_SCHEME}://{AJV_VALIDATOR_HOST}:{AJV_VALIDATOR_PORT}/validate"
    )
else:
    AJV_VALIDATOR_URL = f"{AJV_VALIDATOR_URL}/validate"

Reply:
Yeah happy with this logic. I shall push it now 👍